modules/programs/aerc.nix

```
@@ -102,64 +109,87 @@ in {
         (generators.toKeyValue { inherit mkKeyValue; } global)
         (generators.toINI { inherit mkKeyValue; } local)
       ];
-    mkINI = conf: if builtins.isString conf then conf else toINI conf;
+
+    mkINI = conf: if isString conf then conf else toINI conf;
+
     mkStyleset = attrsets.mapAttrs' (k: v:
-      let value = if builtins.isString v then v else toINI { global = v; };
+      let value = if isString v then v else toINI { global = v; };
       in {
         name = "aerc/stylesets/${k}";
         value.text = joinCfg [ header value ];
       });
+
     mkTemplates = attrsets.mapAttrs' (k: v: {
       name = "aerc/templates/${k}";
       value.text = v;
     });
-    accountsExtraAccounts = builtins.mapAttrs accounts.mkAccount aerc-accounts;
-    accountsExtraConfig =
-      builtins.mapAttrs accounts.mkAccountConfig aerc-accounts;
-    accountsExtraBinds =
-      builtins.mapAttrs accounts.mkAccountBinds aerc-accounts;
-    joinContextual = contextual:
-      with builtins;
-      joinCfg (map mkINI (attrValues contextual));
+
+    accountsExtraAccounts = mapAttrs accounts.mkAccount aerc-accounts;
+
+    accountsExtraConfig = mapAttrs accounts.mkAccountConfig aerc-accounts;
+
+    accountsExtraBinds = mapAttrs accounts.mkAccountBinds aerc-accounts;
+
+    joinContextual = contextual: joinCfg (map mkINI (attrValues contextual));
+
+    recursivelyEmpty = x:
```

```suggestion
    isRecursivelyEmpty = x:
```
IMO this makes more sense, but it's up to you.

---

Thanks, its definitely more sensible.